app/views/layouts/_header.html.erb

```diff
@@ -4,6 +4,13 @@
     <ul>
       <li><%= link_to 'Upcoming Events', upcoming_events_path %></li>
       <li><%= link_to 'New Presentation', new_presentation_path %></li>
+      <li>
+        <% if current_person %>
+          <%= link_to 'Sign Out', signout_path %>
+        <% else %>
+          <%= link_to 'Sign In', '/auth/google_oauth2' %>
```

Comment:
@graygilmore has expressed some concerns with the url being hardcoded. The omniauth gem doesn't expose a routes when I run `rake routes`. Using a hardcoded url seems to be the adopted best practice as railscasts, richonrails and more tutorials are using this method.